api/bibliotheca.py

```diff
@@ -1392,7 +1440,7 @@ def _intrinsic_start_time(self, _db):
         # We don't use Monitor.timestamp() because that will create
         # the timestamp if it doesn't exist -- we want to see whether
         # or not it exists.
-        default_start_time = utc_now() - self.DEFAULT_START_TIME
+        default_start_time = self.DEFAULT_START_TIME
```

Comment:
Previously the default start time was a certain number of years in the past. Now it's a specific date: January 1, 2014.